config.rb

```diff
@@ -40,3 +40,5 @@
 activate :livereload
 activate :neat
 activate :syntax
+
+page '/feed.xml', layout: false
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.